cabal-install/Distribution/Client/Init.hs

```diff
@@ -155,11 +170,44 @@ getPackageName flags = do
   guess    <-     traverse guessPackageName (flagToMaybe $ packageDir flags)
               ?>> Just `fmap` (getCurrentDirectory >>= guessPackageName)
 
+  let verbosity = normal
+  let globalFlags = defaultGlobalFlags
+  let infoFlags = InfoFlags { infoVerbosity  = toFlag normal, infoPackageDBs = [] }
+  (_useSandbox, config) <- loadConfigOrSandboxConfig verbosity globalFlags mempty
+  let configFlags' = savedConfigureFlags config
+      configFlags  = configFlags' {
+        configPackageDBs = configPackageDBs configFlags'
+                           `mappend` infoPackageDBs infoFlags
+        }
+      globalFlags' = savedGlobalFlags    config `mappend` globalFlags
+  (_, _, conf) <- configCompilerAuxEx configFlags
+
+  srcPkgDb <- getSourcePackages verbosity (globalRepos globalFlags')
```

Comment:
Is there a reason we're using the `getSourcePackages` from `Distribution.Client.IndexUtils` rather than the one from `Distribution.Simple.Configure`. The latter is the standard high level wrapper.